packages/dara-components/dara/components/common/stack.py

```diff
@@ -137,5 +137,5 @@ class Stack(LayoutComponent):
     scroll: bool = False
 
     # Dummy init that just passes through arguments to superclass, fixes Pylance complaining about types
-    def __init__(self, *args: ComponentInstance, **kwargs):
+    def __init__(self, *args: Union[ComponentInstance, None], **kwargs):
```

Comment:
Do we need to add this union to other components as well? e.g. card, grid and others that accept ComponentInstance?

Reply:
Yep, I missed those two, thanks